rules-tests/TypeDeclaration/Rector/ClassMethod/AddVoidReturnTypeWhereNoReturnRector/Fixture/skip_never_method.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddVoidReturnTypeWhereNoReturnRector\Fixture;
+
+final class ExitMethod
+{
+    protected function getValues()
```

Comment:
when not yet supported, add `void` is fine imo, than use `ReturnNeverTypeRector` when feature available, the `void` will be changed to `never`, see

https://getrector.com/demo/b32ecefb-b0b4-40ff-be70-c8c2bd5eea5b

Reply:
I added this test-case because my interpretation of

https://github.com/rectorphp/rector-src/blob/c17181453c5493f1ef2e43206759f12d715c18df/rules/TypeDeclaration/TypeInferer/SilentVoidResolver.php#L143-L146

was that we don't want to add `void` when a never type is contained